include/classes/DAO/GuidelinesDAO.class.php

```diff
@@ -51,7 +51,6 @@ public function Create($userID, $title, $abbr, $long_name, $published_date, $ear
 		$long_name = trim($long_name);   // $addslashes is not necessary as it's called in LanguageTextDAO->Create()
 		$earlid = $addslashes(trim($earlid));
 		$preamble = $addslashes(trim($preamble));
-		if ($published_date == '' || is_null($published_date)) $published_date = '0000-00-00';
```

Comment:
Shall this line still be kept as `if ($published_date == '') $published_date = NULL;` to ensure the initial date would be set to `NULL` when the date value is not provided?

Reply:
Thanks, will work on it

Reply:
The column that accepts the published_date already have a default value of NULL, but will still implement this.

Reply:
![screenshot from 2018-05-01 20-44-08](https://user-images.githubusercontent.com/15857679/39490367-90d5f866-4d80-11e8-88a1-4d3e5b349118.png)

Reply:
The concern is, if the `public_date` value provided to this function is empty, the empty value will then be inserted and override the default `NULL`.

Reply:
Ok, now I understand, thanks, I have made the required changes